test/e2e-common.sh

```diff
@@ -143,25 +143,20 @@ function verify_log_access_enabled() {
 function uninstall_pipeline_crd() {
   echo ">> Uninstalling Tekton Pipelines"
   ko delete --ignore-not-found=true -R -f config/
-
-  # Make sure that everything is cleaned up in the current namespace.
-  delete_pipeline_resources
 }
 
 function uninstall_pipeline_crd_version() {
   echo ">> Uninstalling Tekton Pipelines of version $1"
   kubectl delete --ignore-not-found=true -f "https://github.com/tektoncd/pipeline/releases/download/$1/release.yaml"
 
-  if [ "${PIPELINE_FEATURE_GATE}" == "alpha" ]; then
+  if [ -v ${PIPELINE_FEATURE_GATE+x} && "${PIPELINE_FEATURE_GATE}" == "alpha" ]; then
```

Comment:
can this be instead? it seems "+x" is not needed in bash 4.3+?

```suggestion
  if [ -v ${PIPELINE_FEATURE_GATE} && "${PIPELINE_FEATURE_GATE}" == "alpha" ]; then
```

https://www.gnu.org/software/bash/manual/bash.html

Reply:
good question; can you point me to where in that link you're seeing this? (I'm looking at [this section](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion).) I poked through our test runner [dockerfile](https://github.com/tektoncd/plumbing/blob/main/tekton/images/test-runner/Dockerfile) but there's not really a good way for me to tell what version of bash we are using, so I'd prefer to keep it. (Not sure how to go from hash-> dockerfile.)